.github/workflows/build-deploy-dev.yml

```diff
@@ -5,42 +5,133 @@ on:
     types: [opened, reopened, synchronize, closed]
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      bumb:
+        description: "The type of semantic version increment to make. By default it is 'prerelease'."
+        required: false
+        default: "prerelease"
+        type: choice
+        options:
+          - major
+          - premajor
+          - minor
+          - preminor
+          - patch
+          - prepatch
+          - prerelease
+      preId:
+        description: "Identifier to be used to prefix version increments. By default it is 'prerelease' for pull request event and workflow dispatch will follow the last release tag prefix, if any."
+        required: false
+        type: choice
+        options:
+          -
+          - premajor
+          - preminor
+          - prepatch
+          - prerelease
+      releaseType:
+        description: "Select the release type. By default only tag will be created. if you want to create a release with the tag select the type."
```

Comment:
Please adjust the sentence as below:
"Select the release type. By default, only a tag will be created. If you want to create a release with the tag select the type."